### ISSUE TITLE
Chrome DevTools Issue: No label associated with a form field

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/geometric-optics#478]
Originally reported by @Nancy-Salpepi in https://github.com/phetsims/geometric-optics/issues/476 ...

Chrome DevTools has identifed 13 DOM elements as "opportunity for improvement".  They are all of the form:

`<label data-focusable="false" class="a11y-pdom-element`

To reproduce:

1. Run geometric-optics from phetmarks.
2. Open Chrome DevTools and click on the "Issues" tab.  You'll see an Issue that looks like this. Note that the blue icon means severity "An improvement issue: there is an opportunity to improve the page".

<img width="561" alt="screenshot_2553" src="https://github.com/phetsims/geometric-optics/assets/3046552/9777cda6-6bdd-4821-aa12-4170cb43c5de">

3. Expand the issue, like this:

<img width="698" alt="screenshot_2549" src="https://github.com/phetsims/geometric-optics/assets/3046552/61f0ee3b-8e10-4ccf-a425-340be4f331ca">

4. Expanding the "> 13 resources", like this:

<img width="716" alt="screenshot_2550" src="https://github.com/phetsims/geometric-optics/assets/3046552/9856ad33-0929-494e-9a92-f14e83fa3b66">

5. Select the first "Violating node". You will see this in the "Elements" tab:

<img width="530" alt="screenshot_2551" src="https://github.com/phetsims/geometric-optics/assets/3046552/e406f544-9a94-4ca0-9441-abad1097c008">

And in fact every "Violating node" is `<label data-focusable="false" class="a11y-pdom-element`. So there's something going on with a11y and the PDOM here.

**Comment by pixelzoom** [comment]
Questions for @jessegreenberg:

1. Any idea what's going on here?
2. Any idea why this seems specific to Geometric Optics? Perhaps some PDOM feature that the sim is using?
3. Do we need to do anything about this?

**Comment by jessegreenberg** [comment]
There are no `for`  attribute on those `input` elements. In scenery, we don't set the `for` attribute for elements in the PDOM until we try to give the label some inner content. I don't know why that is. This is happening in geometric-optics because the sim has PDOM elements without any content for screen readers. Ill look into https://github.com/phetsims/scenery/issues/1561 to set the `for` attribute faster so there are no audit warnings.

This will have no impact on simulation behavior, you can definitely ignore this.
